framework/contracts/account/manager/src/error.rs

```diff
@@ -88,7 +88,7 @@ pub enum ManagerError {
     #[error("invalid configuration action, {}", error)]
     InvalidConfigAction { error: StdError },
 
-    #[error("Must use SetOwner to change owner")]
+    #[error("Must use ProposeOwner to change owner")]
     MustUseSetOwner {},
```

Comment:
Should we also rename the variant itself?

Reply:
Sure, updated!